include/glow/Backends/BackendOptions.h

```diff
@@ -17,6 +17,7 @@
 #define GLOW_BACKENDS_BACKENDOPTIONS_H
 
 #include "llvm/ADT/SmallVector.h"
+#include <map>
```

Comment:
any reason to not use unordered_map or llvm::StringMap?

Reply:
`std::map` is what's used by `LLVM_YAML_IS_STRING_MAP`, and didn't think it'd make sense to convert from what's deserialized into `std::map` into one of those others.